core/domain/exp_services.py

```diff
@@ -893,6 +894,89 @@ def publish_exploration_and_update_user_profiles(committer, exp_id):
             contributor, contribution_time_msec)
 
 
+def validate_exploration_for_story(exp, raise_error):
+    """Validates an exploration with story validations.
+
+    Args:
+        exp: Exploration. Exploration object to be validated.
+        raise_error: bool. Whether to raise an Exception when a validation error
```

Comment:
Let's use `strict` as the name instead, that's more common/standard in the code base.

Reply:
Done